pyclam/manifold.py

```diff
@@ -412,7 +452,7 @@ def __eq__(self, other: 'Graph') -> bool:
                 set(self.edges[cluster]) == set(other.edges[cluster])
                 for cluster in self.clusters
             ))
-        return cluster_equality and edges_equality
+            return cluster_equality and edges_equality
```

Comment:
looks like weird spacing in the diff

Reply:
I did that because of some Rust instincts translating over into my python. The tab makes that a proper, exhaustive if-else statement.